packages/core/src/components/modal/modal.scss

```diff
@@ -0,0 +1,79 @@
+.modal {
```

Comment:
You'll need to add some styles for the `:host` element itself. Things like `box-sizing`, `display`. Take a look at `button` or `tag` on the `main` branch

Reply:
@Drakodin bump^